Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -939,3 +940,31 @@
+  id: UplinkHardsuitSyndicateEliteModsuit
+  name: uplink-modsuit-elitemodsuit-name
+  description: uplink-modsuit-elitemodsuit-desc
+  icon: { sprite: /Textures/_Goobstation/Clothing/OuterClothing/Modsuits/syndicateelitemodsuit.rsi, state: chestplate }
+  productEntity: EliteModsuitBackpack
+  discountCategory: rareDiscounts
+  discountDownTo:
+    Telecrystal: 30
+  cost:
+    Telecrystal: 60
+  categories:
+  - UplinkWearables
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)